doc/source/getting-started/administration.rst

```diff
@@ -92,54 +92,59 @@ The following checks are required when performing the legal review of the projec
 
 .. card:: |uncheck| The project contains the right licensing.
 
-    * The project contains the right license.
+    * The project has the correct license.
     * The contribution does not contain any strong encryption.
     * Ansys official logos and branding images are used in the project.
-    * The Ansys copyright appears in the right location as required by the Legal department.
-    * The copyright has the right formatting, which is ``Copyright (C) YYYY ANSYS, Inc.``.
-    * The contribution does not embody any Ansys intellectual property that is not approved for open sourcing.
-    * The contribution does not embody any invention for which Ansys has sought or received patent protection.
-    * Any third-party open sources included in the contribution have been reviewed for security vulnerabilities and have had their license files included in the repository.
-
-Open source dependencies that are not distributed as part of the project do not
-need their licenses included in the Ansys repository. Examples include
-dependent Node package manager (``npm``) modules or Python packages from PyPI.
+    * The Ansys copyright appears in the correct location as required by the
+      Legal department.
+    * The copyright has the proper formatting, which is ``Copyright (C) YYYY ANSYS, Inc.``.
+    * The contribution does not embody any unapproved Ansys intellectual
+      property for open sourcing.
+    * The contribution does not embody any inventions for which Ansys has
+      sought or received patent protection.
+    * Any third-party open-source code included in the contribution has been
+      reviewed for security vulnerabilities and includes their license files in
+      the repository.
+
+Open-source dependencies not distributed as part of the project do not need
```

Comment:
```suggestion
Open source dependencies not distributed as part of the project do not need
```
Seems like the checker here doesn't like the absence of the hyphen, but we are following Google style guidelines.